lib/ui/hooks.dart

```diff
@@ -65,11 +65,11 @@ String _localeClosure() {
 
 @pragma('vm:entry-point')
 // ignore: unused_element
-_LocaleClosure _getLocaleClosure() => _localeClosure;
+_LocaleClosure/*?*/ _getLocaleClosure() => _localeClosure;
```

Comment:
I think this is safe to be non-null. `_localeClosure` is just a tear-off of a top-level function.